### ISSUE TITLE
get 68% accuracy on restaurant dataset

### ISSUE THREAD

**Issue by gliu9966** [lpq29743/IAN#10]
How do you solve the overfitting problem？My experimental accuracy is always low

**Comment by lpq29743** [comment]
I'm sorry for the late reply. Have you solved the problem you met? Could you give a clear description.
As far as I know, dropout and early stopping can ease the over-fitting problem to some extent.

**Comment by Shengguowei** [comment]
yes,my score is low too.I use your code, preprocess text,and then set pre_processed = 0 ,then run main(),and in laptop data set ,it's socre is low,in train is about 0.637587,and in test is about 0.642516, this is so low, can you give me a reply quickly（if your the author of the paper),In ABSA-keras, the author of it ,alse find your model's result is not more than paper announce……I think you may forget something.

**Comment by studykz** [comment]
@gliu9966 I want to ask you a question, what is your version of tensorflow running the codes? I meet a question that "AttributeError: module 'tensorflow' has no attribute 'keras'"
